spec/double_entry_spec.rb

```diff
@@ -409,12 +409,6 @@
       end.to raise_error DoubleEntry::AccountScopeMismatchError
     end
 
-    it 'raises exception if you try to transfer between the same account, despite it being scoped' do
-      expect do
-        DoubleEntry.transfer(Money.new(10_00), :from => ryans_cash, :to => ryans_cash, :code => :xfer)
-      end.to raise_error(DoubleEntry::TransferNotAllowed)
-    end
-
```

Comment:
This test was removed here:  https://github.com/envato/double_entry/pull/144/commits/2a98dfbc81b669c7460f62a9f9136726d9f496f7

Though I sense it would still be good for this error to be raised if the identifier `from` and `to` are the same since the assumption is that this value is a unique id (or uuid?)